File: Loiane_Training/Java_Basico/Exer Aula 14_15/Ex17.java
import java.util.Scanner;

class Ex17 {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		System.out.println("Digite um ano");
		int ano = scan.nextInt();

		if (ano % 4 == 0) {
			System.out.println("Ano bissexto");
		} else {
			System.out.println("Ano nao bissexto");
		}
		scan.close();
	}
}

File: loiane-training-csharp/procedural-poo/c/Ex17.cs
using System;

public class Ex17
{
    static void Main(string[] args)
    {
        Console.WriteLine("Digite um ano para verificar se e Bissexto");
        int ano = int.Parse(Console.ReadLine());

        if (ano % 4 == 0) {
            Console.WriteLine("Ano bissexto!");
        } else
        {
            Console.WriteLine("Ano nao bissexto!");
        }
    }
}
